pkg/util/timeutil/zoneinfo.go

```diff
@@ -30,7 +30,14 @@ func LoadLocation(name string) (*time.Location, error) {
 	switch loweredName {
 	case "local", "default":
 		loweredName = "utc"
-		name = "UTC"
+	case "EST5EDT":
```

Comment:
i am not following how this works - wouldn't the call to `strings.ToLower` above mean that this should be `est5edt` instead?